bigbluebutton-html5/imports/ui/components/presentation/service.js

```diff
@@ -95,6 +95,11 @@ const parseCurrentSlideContent = (yesValue, noValue, abstentionValue, trueValue,
   const pollRegex = /[1-9A-Ia-i][.)].*/g;
   let optionsPoll = safeMatch(pollRegex, content, []);
   const optionsWithLabels = [];
+
+  if (hasYN) {
+    optionsPoll = ['Yes', 'No'];
```

Comment:
Does this interfere with YN polls in other languages?

Reply:
Updated to use formatted labels